utils/include/XMLParser.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "pugixml.hpp" //namespace pugi{ class xml_document; } didn't work
+
+class XMLParser
+{
+public:
+	XMLParser() = default;
```

Comment:
Why don't you want to implement ctor that calls your `Initialize()` method immediately?